src/mocking/accessors/filtering.ts

```diff
@@ -80,11 +80,11 @@ const createPredicate = (key: string, value: any, testData: GenericData): Predic
 
 export const genericSort = <TData extends GenericData>(data: TData[], variables: SortingArgs): TData[] => {
   const { orderBy } = variables
-  if (!orderBy) {
+  if (!orderBy?.length) {
     return data
   }
 
-  const [field, direction] = orderBy.split('_')
+  const [field, direction] = orderBy[0].split('_')
```

Comment:
I'm a little confused why `orderBy` is now array? This works for now because we are passing only one element to `orderBy` everywhere, but I assume it's probably wrong.

Reply:
My guess is it became an array so we can do multiple rules for sorting. So if the first rule places the items in the same position, you can use the second to determine the final order

Reply:
But for mocking we can safely just use the first one I think